blog/2010/10/clarifications-about-call-for-instructor-training.html

```diff
@@ -0,0 +1,107 @@
+<p>
+Since we announce
+<a href="{{page.root}}/blog/2015/10/call-for-instructor-training.html">that the applications for December Instructor Training</a>
+we receive a lot of questions. We amend the announcement with the answer to some
+questions and we believe that others need a post dedicated to them.
+</p>
+<!-- end excerpt -->
+<p>
+We are been excited to see the strong interest in this program
+and the enthusiasm for getting even larger groups together for this training.
+Due to this interest we wanted to provide some updates.
+</p>
+<p>
+First, this program is a pilot for delivering
+training to groups at a larger scale.
+We should have make this more clear when we first announced
+<a href="{{page.root}}/blog/2015/09/rebooting-instructor-training.html">the instructor training reboot</a>
+and we are sorry about that.
+This pilot
+</p>
+<ol>
+    <li>
+        <p>
+        Satisfies pent-up demand for IT in our community.
+        </p>
+        <p>
+        We've made commitments to Partners and Affiliates to provide training
+        and help build communities of instructors at their institutions
+        <strong>but we also want to do this effectively</strong>
+        so we will ask you to be a little more patient if you are not selected for this round.
+        If you have questions, <a href="mailto:execdir@software-carpentry.org">Jonah</a> will be happy to answer them.
+        </p>
+    </li>
+    <li>
+        <p>
+        Allows us to compare two different approaches to open online training
+        and get a handle on their true costs.
+        </p>
+    </li>
+    <li>
+        <p>
+        Allows us to see what effect the requirement for preparatory work has
+        on completion and follow-through rates.
+        </p>
+    </li>
+    <li>
+        <p>
+        Provides practical training for new trainers.
+        </p>
+    </li>
+    <li>
+        <p>
+        Provides raw material that those new trainers can turn into reusable
+        IT curriculum.
+        </p>
+    </li>
+</ol>
+<p>
+Some points of clarification from
+<a href="{{page.root}}/blog/2015/10/call-for-instructor-training.html">FAQs</a>:
+<ul>
+    <li>
+        <strong>Group consideration:</strong>
+        For this pilot,
+        we will continue to consider all groups equally,
+        so we know how this format works for different organizations.
+        <strong>
+            In the next round of instructor training,
+            Partners and Affiliates will be prioritized,
+            and based on the outcome and effectiveness of this approach,
+            it will be delivered either as an individual online version or in this group format.
+        </strong>
+    </li>
+    <li>
+        <strong>Group size restrictions:</strong>
+        <a href="{{page.root}}/blog/2015/09/rebooting-instructor-training.html">At first we said</a>
+        "we will invite groups of 4-6 people to take part in the two-day training course in December 2015"
```

Comment:
When we first though about group sizes for this pilot, our concerns were mainly that we would end up with groups that were very small. The interest that this pilot has garnered caught us a bit by surprise. Hence we are now having to restrict this pilot to 10 people from each organization, with a maximum group size of 5. This is mainly to ensure that we are able to give high quality instruction to all involved, that the students have a good chance at teaching at the subsequent workshop, and that we will be able to teach people at multiple institutions.